packages/frontend-language-toggle/README.md

```diff
@@ -0,0 +1,160 @@
+<!-- Improved compatibility of back to top link: See: https://github.com/othneildrew/Best-README-Template/pull/73 -->
+
+<a name="readme-top"></a>
+
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  
+<h3 align="center">GOV UK One Login Language Toggle</h3>
+  <p align="center">
+    This package enables GOV UK LOGIN frontend Node.js applications to add a language toggle.
+    <br />
+    <a href=""><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/govuk-one-login/di-fec-ga4-demo">View Demo</a>
+    ·
+    <a href="https://github.com/govuk-one-login/di-fec-ga4-demo/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/govuk-one-login/di-fec-ga4-demo/issues">Request Feature</a>
+  </p>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#installation">Installation</a></li>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+      </ul>
+    </li>
+  </ol>
+</details>
+
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+
+The GDS One Login Language toggle node package is a shared, reusable nunjuck component created to facilitate the integration of a language toggle.
+
+The purpose of this component is to make it as easy as possible for the various pods that make up the One Login journey to configure and add a configurable language toggle.
+
+The package is owned by the DI Frontend Capability team, part of the development of this tool involves ongoing discovery with the pods responsible for maintaining the frontend repositories that make up the One Login journey. As more information is collated, the package and documentation will be updated. As such, it is considered a WIP and the pods will be notified when a stable release is ready.
```

Comment:
Depending on the outcome of the Go/No Go call tomorrow, we should update this section to include info about the stable releaase